src/components/AuthorizationWindow/AuthorizationWindow.jsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useContext, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Header } from '../index';
+import { Context } from '../..';
+import logo from '../../img/logo.png';
+import './style.scss';
+
+const AuthorizationWindow = () => {
```

Comment:
why do you need 2 components for authorization and registration? Delete unnecessary